airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/spec.json

```diff
@@ -90,6 +90,13 @@
         "order": 2,
         "type": "string",
         "format": "date"
+      },
+      "attribution_window": {
+        "title": "Attribution Window",
+        "description": "The attribution window in days.",
+        "default": 0,
+        "order": 4,
```

Comment:
Can be order 3 ?

Reply:
Since the `report_granularity` is gone now, I'll adjust this.